src/citrine/_serialization/properties.py

```diff
@@ -340,6 +340,26 @@ def _serialize(self, value: list) -> list:
             serialized.append(self.element_type.serialize(element))
         return serialized
 
+    def __set__(self, obj, value: typing.Union[SerializedType, DeserializedType]):
```

Comment:
I added this just for the List Property for now, but this should be generalized for the other collections